packages/jsii-rosetta/bin/jsii-rosetta.ts

```diff
@@ -197,30 +215,26 @@ function main() {
         // compilerhost. Have to make all file references absolute before we chdir
         // though.
         const absAssemblies = (args.ASSEMBLY.length > 0 ? args.ASSEMBLY : ['.']).map((x) => path.resolve(x));
-        const absOutput = path.resolve(args.output);
-        const absCache = fmap(args['cache-from'], path.resolve);
+
+        const absCacheFrom = fmap(args.cache ?? args['cache-from'], path.resolve);
+        const absCacheTo = fmap(args.cache ?? args['cache-to'] ?? args.output, path.resolve);
```

Comment:
Did you consider adding `.conflicts('output', ['cache-to', 'cache'])` instead of ignoring `output` if it is specified along with the cache arguments?

Reply:
We have `.conflicts('cache-to', 'output')`. Is that not good enough?